client/signup/config/flows-pure.js

```diff
@@ -113,7 +114,9 @@ export function generateFlows( {
 			steps: isEnabled( 'signup/professional-email-step' )
 				? [ 'user', 'domains', 'emails', 'plans' ]
 				: [ 'user', 'domains', 'plans' ],
-			destination: getSignupDestination,
+			destination: isEnabled( 'signup/setup-site-after-checkout' )
```

Comment:
I think it'd be better to check `isEnabled( 'signup/setup-site-after-checkout' )` within the body of `getSignupDestination()`. That's because IMO we want the design picker to appear for flows other than `onboarding`. The `premium` flow for instance is accessed with the `/start/premium` url, and is basically the `onboarding` flow except the premium plan gets auto selected. It means when a logged out visitor selects the plan they want on https://wordpress.com/pricing/ it'll auto choose that plan. In that case we want the design picker to appear after the checkout.

Reply:
Does every flow using `getSignupDestination` need to show the design picker after checkout? If so, it's good to check it within `getSignupDestination`

Reply:
The only flows that gives me pause are the ecommerce ones. I'll test now but I have a feeling that they're already not following the usual `getSignupDestination()` anyway since I'm pretty sure they don't go to `/home` when done.

Reply:
Yes the ecommerce flows are already ignoring what the `getSignupDestination` function returns, so I think every flow using `getSignupDestination` should be told to go the design picker. The ecommerce flows will continue to ignore it with whatever mechanism it's currently using.

Reply:
Sounds good 👍